first_assesment/repository/pokemon_repository.go

```diff
@@ -0,0 +1,56 @@
+package repository
+
+import (
+	"errors"
+	"os"
+
+	"first/model"
+
+	"github.com/gocarina/gocsv"
+)
+
+type PokemonRepository struct {
+	file string
+}
+
+func NewPokemonRepository() (*PokemonRepository, error) {
+	return &PokemonRepository{
+		file: "csv/input_file.csv",
+	}, nil
+
+}
```

Comment:
What if you change your file route?

It's better for this function to have the route as a parameter, and read this value again from a config file.
```suggestion
func NewPokemonRepository(csvFileRoute string) (*PokemonRepository, error) {
	return &PokemonRepository{
		file: csvFileRoute,
	}, nil
}
```

Doing it this way, you could also give it use to your error variable, wich always returns nil.

e.g. if your file is empty, or doesn't end with `.csv`.

Reply:
fix it, i prefer that the repository get the path (from config file) inside the repository.